packages/compass-data-modeling/src/components/new-diagram/new-diagram-modal.spec.tsx

```diff
@@ -0,0 +1,410 @@
+import React from 'react';
```

Comment:
Most of the code in this file is from `new-diagram-form.spec.tsx`, moved it to this folder